libraries/aws_iam_root_user.rb

```diff
@@ -20,6 +20,16 @@ def has_mfa_enabled?
     summary_account['AccountMFAEnabled'] == 1
   end
 
+  def has_virtual_mfa_devices?
```

Comment:
I think this should probably be named `has_a_virtual_mfa_device?`.  While the API is plural, the console and [docs](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_mfa_enable_virtual.html) state only one MFA device may be associated with a user.